engineering_tools/carma

```diff
@@ -116,22 +151,44 @@ carma-config__reset() {
 }
 
 carma-config__list_local() {
+    if [ "$1" = "-d" ]; then
```

Comment:
This'd probably be best not as an option but as a sequential thing, i.e. run `docker images usdotfhwastol/carma-config` then run `docker images usdotfhwastoldev/carma-config` then `docker images usdotfhwastolcandidate/carma-config` whenever this gets called. That way it really is listing all local images.

Reply:
I'd still suggest we make this list **all** locally available images, so basically just list the usdotfhwastol usdotfhwastoldev and usdotfhwastolcandidate images in sequence. This avoids confusion about which images are actually installed locally and available. I think leaving the flags for the list remote is fine (since it makes sense to kind of hide the pre-release images), but they should just always apply here.